src/activities/jumbled_words/Jumbled_words.qml

```diff
@@ -0,0 +1,403 @@
+                }
+                font {
+                    pointSize: (40 + ApplicationSettings.baseFontSize) * ApplicationInfo.fontRatio
+                    capitalization: ApplicationSettings.fontCapitalization
+                    weight: Font.Bold
+                    family: GCSingletonFontLoader.fontLoader.name
+                    letterSpacing: ApplicationSettings.fontLetterSpacing
+                    wordSpacing: 10
+                }
+
+                cursorDelegate: Rectangle {
+                    id: cursor
+                    width: 10
+                    height: parent.cursorRectangle.height
+                    color: 'white'
+                    SequentialAnimation on opacity {
+                        running: true
+                        loops: Animation.Infinite
+                        PropertyAnimation {
+                            to: 0.2
+                            duration: 1000
+                        }
+                        PropertyAnimation {
+                            to: 1
+                            duration: 1000
+                        }
+                    }
+                }
+            }
+        }
+
+        Rectangle {
+            id: submit
+            anchors.fill: submitText
+            border.color: "#FFFFFFFF"
+            border.width: 2
+            height:submitText.height
+            color: "#000065"
+            opacity: 0.31
+            radius: 10
+
+            MouseArea {
+                id: mouseAreaSubmit
+                anchors.fill: parent
+                hoverEnabled: ApplicationInfo.isMobile ? false : true
+                onClicked: {
+                    if(Activity.checkAnswer())
+                    {
+                        Activity.showHint()
+                    }
+                    edit.text=""
+                 }
+            }
+        }
+
+        GCText {
+            id: submitText
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors{
+                bottom: bar.top
+                margins: 10
+            }
+            opacity: 1.0
+            text:qsTr("Submit")
+            fontSize: mediumSize
+            font.bold: true
+            style: Text.Outline
+            styleColor: "#2a2a2a"
+           color: "white"
+        }
+
+        Score {
+            id: score
+            anchors.top: parent.top
+            anchors.topMargin: 10 * ApplicationInfo.ratio
+            anchors.left: parent.left
+            anchors.leftMargin: 10 * ApplicationInfo.ratio
+            anchors.bottom: undefined
+            anchors.right: undefined
+        }
+
+        Bar {
+            id: bar
+            content: BarEnumContent { value: help | home | level | reload | config}
+            onHelpClicked: {
+                displayDialog(dialogHelpLeftRight)
+            }
+            onHomeClicked: activity.home()
+            onReloadClicked: {
+               edit.text = Qt.binding(function() { return ""})
```

Comment:
binding are supposed to be used with variables that can change not with static value. In this case, it is not useful, you can remove them